### DISCUSSION TITLE
question: why golangci-lint with option --fix does not trigger any fix from testifylint?

### DISCUSSION THREAD

**Discussion by peczenyj** [Antonboom/testifylint#184] [Q&A] [answered]
Hello

I had to use the program `testifylint` with option `-fix` on my repositories, since `golangci-lint --fix` somehow does not trigger the automatic fix when possible

My question is: why?

this linter seems to support the fix option. Perhaps the interface with golangci-lint need some adaptation?

I'd like to avoid use multiple tools, since golangci-lint can centralize all code needed

**Comment by Antonboom** [comment, marked as answer]
Hi!

Because golangci-lint uses their own fixing approach, some linters supports it but some not.
We are awaiting of https://github.com/golangci/golangci-lint/issues/1779
